android/src/main/java/bbct/android/common/activity/MainActivity.java

```diff
@@ -69,18 +70,17 @@ public void onCreate(Bundle savedInstanceState) {
                     BaseballCardContract.PROJECTION, null, null, null);
 
             FragmentTransaction ft = this.getSupportFragmentManager().beginTransaction();
-            if (cursor == null || cursor.getCount() == 0) {
-                ft.add(R.id.fragment_holder, new BaseballCardDetails(), FragmentTags.EDIT_CARD);
-            } else {
-                ft.add(R.id.fragment_holder, new BaseballCardList(), FragmentTags.CARD_LIST);
-                cursor.close();
-            }
+            ft.add(R.id.fragment_holder, new BaseballCardList(), FragmentTags.CARD_LIST);
```

Comment:
Should you also call `addToBackStack()` here?

Reply:
I think I should.